src/components/Form/Dropdown.js

```diff
@@ -2,31 +2,28 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { ButtonGroup, DropdownButton, MenuItem, noop } from 'patternfly-react';
 
-export const Dropdown = ({ id, value, disabled, onChange, onBlur, choices }) => (
-  <ButtonGroup justified key={id}>
-    <DropdownButton
-      id={id}
-      bsStyle="default"
-      className="kubevirt-dropdown"
-      title={value}
-      onSelect={onChange}
-      onBlur={onBlur}
-      disabled={disabled}
-    >
-      {choices.map(choice => {
-        const isObject = typeof choice === 'object';
-        const key = isObject ? choice.id || choice.name : choice;
-        const val = isObject ? choice.name : choice;
+export const Dropdown = ({ id, value, disabled, onChange, onBlur, choices }) => {
```

Comment:
I am not in a favor of this approach. This will create unnecessary amount of callback handlers. Can't we rather lookup the name somewhere higher, e.g. in `request.js`?

Reply:
apparently we can pass object to MenuItem's eventKey so Im passing choice as whole. With this I can keep onSelect on DropdownButton.

Reply:
nice